### ISSUE TITLE
[Question]: Why do I need to delete and rebuild the index every time you perform a full-text search?

### ISSUE THREAD

**Issue by xxzhang0927** [infiniflow/infinity#1620]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Version or Commit ID

v0.3.0

### Actual behavior and How to reproduce it

def add_index():
    table.drop_index("my_index")
    # 索引是现场创建的
    res = table.create_index(
            "my_index",
            [
                infinity.index.IndexInfo("content", infinity.index.IndexType.FullText, [infinity.index.InitParameter("ANALYZER", "chinese")]),
            ],
            infinity.common.ConflictType.Error,
        )
    
def query_index():
    res = table.show_index("my_index")
    print(res)
    
def query_full_text():
    # question = "间接服务角色"
    question = "社会工作者 小燕 肢体障碍人士 就业 创业 成长计划 项目执行 资源筹措者 角色 反思 重建自信 创业园区 咨询服务 微信公众号"
    # question = "社会工作者小燕肢体障碍人士就业创业成长计划项目执行资源筹措者角色反思重建自信创业园区咨询服务微信公众号"
    # question = "10岁女孩小丽，父亲因为犯罪行为被关进监狱，母亲在她很小的时候就离家出走，她由爷爷奶奶照顾。一年前爷爷去世，奶奶72岁，脚残疾，小丽身体状况很差，经常生病，学习困难，性格内向，经常被同学欺负。她从不会把这些情况告诉老师，怕给老师惹麻烦，社会工作者可以运用理论来帮助这个小女孩。\nA.社会支持,B.人本主义,C精神分析,D.认知行为"
    qb_result = (
        table.output(["content", "_score"])
        .match("content",question,)
        .to_pl()
    )
    print(f"question: {question}")
    print(qb_result)

if __name__ == '__main__':
    add_index()
    query_full_text()

If I don't use add_index()
The result is empty

**Comment by yuzhichang** [comment]
Don't need to delete and rebuild the index every time before performing a full-text search.

Here's a sample:
```
if name == 'main':
    add_index()
    query_full_text()
    query_full_text()
    query_full_text()

```

**Comment by xxzhang0927** [comment]
Why do I comment add_index and then execute it after restarting, the result is empty?How do I create a persistent index

**Comment by yuzhichang** [comment]
Create index per the spec, ignore if it already exist:
```
res = table.create_index(
"my_index",
[
infinity.index.IndexInfo("content", infinity.index.IndexType.FullText, [infinity.index.InitParameter("ANALYZER", "chinese")]),
],
infinity.common.ConflictType.Ignore,
)
```

**Comment by xxzhang0927** [comment]
or not 
![image](https://github.com/user-attachments/assets/06d8afbe-2d7b-427b-866a-a6635c460e1f)

**Comment by yangzq50** [comment]
Can you show the full baitong_query.py ?

**Comment by warran2** [comment]
I also have the same problem.

**Comment by JinHai-CN** [comment]
> I also have the same problem.

You can search after creating an index. After restart the server or reconnect infinity instance, you don't need to create the index any more. 

Would you please provide your python scripts, to help me understand your problem?

**Comment by warran2** [comment]
thanks，is had post a new issue

**Comment by yuzhichang** [comment]
Replaced with #1691

**Comment by JinHai-CN** [comment]
Fixed by #1698
